src/panel/panel.vala

```diff
@@ -533,15 +541,38 @@ namespace Budgie {
 								expected_uuids.remove_link(g);
 							}
 							message("Unable to load invalid applet: %s", applets[i]);
+							applet_removed(applets[i]);
+							continue;
+						}
+
+						info = this.add_pending(applets[i], name);
+						if (info == null) {
 							continue;
 						}
-						this.add_pending(applets[i], name);
-						manager.modprobe(name);
+					}
+
+					/* um add this bro to the panel :o */
```

Comment:
Probably not necessary.

Reply:
For the record, that was already there prior to this PR, I just didn't remove it. I'll delete it though.

Reply:
Ah. My bad.